Naming.java

```diff
@@ -0,0 +1,31 @@
+package az.alizeynalli.cleancode.naming;
+
+public class Naming {
+
+    public Date d;
+    public List<Testcase> find(User user){};
+    
+    public Account[] AccountList;
+
+    public void deleteUser(User pUser){};
+    
+    public URL r;
+  
+    public class TableRepresent{};
+    pubcli void car(){};
+  
+    public saveUser(){}; createAccount(){}; generateContract(){};
+
+    public class AccountFactory{}
+    public class AccountObserver{}
+    
+    public class User;
+    public String userName;  
+    public void editUser(User user){};
+
+    public void saveIfUserUnder18(){};
+    public class UserFormRealTimeFiller {};
+    
+    public int daysInYear = 360;
+    public int hoursInWeek = 168;
```

Comment:
// Use constants, enums instead of magic numbers, abbreviations
    // Good
    public enum Calendar {
    DAYS_IN_YEAR(360), 
    HOURS_IN_WEEK(168)    
    }